src/core/webidl.js

```diff
@@ -285,6 +270,24 @@ function renderWebIDL(idlElement) {
     }
     registerDefinition(elem, [title]);
   });
+  // cross reference
+  const closestCite = idlElement.closest("[data-cite], body");
+  const { dataset } = closestCite;
+  if (!dataset.cite) {
+    dataset.cite = "WebIDL";
+    return;
+  }
+  const hasWebIDL = dataset.cite
+    .split(/\s+/)
+    .map(item => item.toLowerCase())
+    .includes("webidl");
+
+  if (hasWebIDL) return;
+
+  dataset.cite = ["WebIDL"]
+    .concat(dataset.cite.split(/\s+/))
+    .filter(i => i)
+    .join(" ");
```

Comment:
How about ` ["WebIDL"].concat(cites).join(" ").trim()` instead of running that `array.filter`?

Reply:
I'd love `["WebIDL", ...cites].join(" ").trim()`

Reply:
>It'll ignore leading and trailing in cites spaces

Not sure I follow, can you give me an example?

Reply:
``` js
datacite = "  a b    c  ";
console.log({ newcite: ["webidl"].concat(datacite.split(/\s+/)).join(" ") })
```

Reply:
``` js
datacite = "  a b    c  ";
console.log({ newcite: ["webidl"].concat(datacite.split(/\s+/)).join(" ") })
```

it'll work if we trim `datacite` too though.

Reply:
@saschanaz I like your suggestions! Forgot about that.

Reply:
Hmm, `` `WebIDL ${cites.join(" ").trim()}` ``?

Reply:
We can do:
``` js
const cites = dataset.cite.trim().toLowerCase().split(/\s+/);
const hasWebIDL  = cites.includes("webidl");
if (hasWebIDL) return;
dataset.cite = ["WebIDL", ...cites].join(" ");
```

Reply:
I don't like the `string.includes` part, there might be a new spec with `webidl` in its name (though unlikely).

Reply:
`/\bwebidl\b/i.match(dataset.cite)` 😺

Reply:
ok, made a mishmash of the above 💃